non_iid_detection/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.22.0
```

Comment:
Add comment that developer version of cleanlab must be installed, see some of the other requirements.txt in this repo like the active learning one for examples